packages/account/src/test-utils/launchNode.ts

```diff
@@ -120,6 +120,9 @@ export const launchNode = async ({
     const poaInstantFlagValue = getFlagValueFromArgs(args, '--poa-instant');
     const poaInstant = poaInstantFlagValue === 'true' || poaInstantFlagValue === undefined;
 
+    const nativeExecutorVersionFlagValue = getFlagValueFromArgs(args, '--native-executor-version');
+    const nativeExecutorVersion = nativeExecutorVersionFlagValue || '0';
+
```

Comment:
Nit - since the final line width is the same. 😆 

```suggestion
    const nativeExecutorVersion = getFlagValueFromArgs(args, '--native-executor-version') || '0';

```